### ISSUE TITLE
Initialise namespace if not exist

### ISSUE THREAD

**Issue by mochekir** [OldBlackJoe/mybatis-mapper#29]
Namespace is overrided if there is two files with same namespace
Is it possible to add this condition before initialise namespace ? 

here is the source code below : 

=> [file and line](https://github.com/OldBlackJoe/mybatis-mapper/blob/b86401eb68fa707b089915d2cd3ecf50921b6002/index.js#L39)

```
if(!myBatisMapper[children.attrs.namespace]) {
      myBatisMapper[children.attrs.namespace] = {};
}
```

Could you please permit the pull-request. 

Thank you

**Comment by mochekir** [comment]
Can you please add this fix ? 
Thank you

**Comment by OldBlackJoe** [comment]
This issue has been fixed and released as version 0.7.1. thank you.
